packages/dialog/src/index.js

```diff
@@ -179,3 +201,27 @@ function useForkedRef(...refs) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, refs);
 }
+
+function validateAriaLabel(props, name, compName, ...rest) {
```

Comment:
Let's call this function `ariaLabelType` in keeping with the idea of "prop types". This function is a "type" of prop.

Reply:
Updated in https://github.com/reach/reach-ui/pull/317/commits/ef3e5eaa00ca3fd34bf41657c0312a6e8d0ab338